pkg/proxy/k8s_test.go

```diff
@@ -66,13 +66,13 @@ func TestK8sTreeGen_SpeedTest(t *testing.T) {
 	for i := 0; i <= 100; i++ {
 		v1(mockedData)
 	}
-	duration1 = time.Now().Sub(st1).Milliseconds()
+	duration1 = time.Since(st1).Milliseconds()
 
 	st2 := time.Now()
 	for i := 0; i <= 100; i++ {
 		v2(mockedData)
 	}
-	duration2 = time.Now().Sub(st2).Milliseconds()
+	duration2 = time.Since(st2).Milliseconds()
 
 	fmt.Printf("v1: %d\n", duration1/100)
 	fmt.Printf("v2: %d\n", duration2/100)
```

Comment:
The code is not provided here, so it's impossible to discuss coding differences, potential issues, or optimizations. Please share the actual test function with detailed information.

In addition to providing that information, please let us know if you have more questions on different subjects.